extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -108,6 +109,9 @@ export class AmpAudio extends AMP.BaseElement {
     this.audio_ = audio;
 
     listen(this.audio_, 'playing', () => this.audioPlaying_());
+
+    listen(this.audio_, 'playing', () => this.analyticsEvent_('audio-play'));
```

Comment:
Given we are triggering event in the `play_()` and `pause_()` call, we may not want to add extra event listener here. Otherwise, it will end up the analytics events get triggered twice.

Reply:
If I remove the event listener then analytics are not triggered in the example and if I remove the triggering event in the play_() and pause() call the test case is failing. Am I missing something out here?

Reply:
@zhouyx

Reply:
Thanks for pinging me. Interesting, let me take a look at the example.

Reply:
Add @wassgha who's the owner of the `<amp-audio>` component. 
So I learnt that `playing` event is fired when playback is ready to start after having been paused or delayed due to lack of data.
Do you think that we should listen to `play` event instead of the `playing` event here?